nextflow.config

```diff
@@ -39,6 +39,10 @@ params {
     split_fastq = 0 // FASTQ files will not be split by default
     save_split_fastqs  = false
 
+    // UMI tagged reads
+    umi_read_structure = null // no umi
+    group_by_umi_strategy = 'Adjacency' //
```

Comment:
If we don't have umi by default, why do we have a strategy to group by?

Reply:
Francesco proposed it as the default version on grouping. I don't know how to set a default value otherwise. It won't be used unless --umi is set